src/Exception/QueryError.php

```diff
@@ -26,14 +31,27 @@ class QueryError extends RuntimeException
     public function __construct($errorDetails)
     {
         $this->errorDetails = $errorDetails['errors'][0];
+        if (\array_key_exists('data', $errorDetails)){
+            $this->data = $errorDetails['data'];
+        } else {
+            $this->data = null;
```

Comment:
This else is redundant, the `$this->data` is by default `null`, no need to set it to `null` if the data key is not present in the `$errorDetails` array.